packages/playground/src/dashboard/bridge_view.vue

```diff
@@ -68,7 +60,6 @@ const qrCodeText = ref("");
 const depositFee = ref(0);
 
 onMounted(async () => {
-  selectedName.value = items.value.filter(item => item.id === selectedItem.value.id)[0].name;
```

Comment:
> this will have an effect on withdraw and deposit dialog; they are using `selectedName`

re-added in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/2135/commits/9daf1ef9bce42cd311cdd728ecca5bc92ea474ae